cmd/vic-machine/delete/delete.go

```diff
@@ -61,6 +61,12 @@ func (d *Uninstall) Flags() []cli.Flag {
 			Usage:       "Compute resource path, e.g. /ha-datacenter/host/myCluster/Resources/myRP",
```

Comment:
"The compute resource containing the Virtual Container Host; requires the 'name' argument be supplied"